Ginger/GingerCoreNET/RunLib/DriverBase.cs

```diff
@@ -173,5 +177,30 @@ public virtual void UpdateContext(Context context)
         {
             BusinessFlow = context.BusinessFlow;
         }
+
+
+        #region VirtualDrivers
+
+        //TODO: .net STandard 2.1 Use C# 8 default interface implmentation  once we upgrade to .net standard2.1
+        /// <summary>
+        /// WIll give list of active virtual drivers
+        /// </summary>
+        public static List<KeyValuePair<string, DriverBase>> VirtualDrivers = new List<KeyValuePair<string, DriverBase>>();
```

Comment:
Codacy found an issue: [Change the visibility of 'VirtualDrivers' or make it 'const' or 'readonly'.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6116697)